README.md

```diff
@@ -66,11 +66,13 @@ You can find above templates and the ones provided by the community in **[this l
 
 ## Requirements
 
-* v12.16+ < Node.js < 15
-* v6.13.7+ < npm < 7
+* v12.16+ < Node.js
+* v6.13.7+ < npm
 
 Install both packages using [official installer](https://nodejs.org/en/download/). After installation make sure both packages have proper version by running `node -v` and `npm -v`. To upgrade invalid npm version run `npm install npm@latest -g`
 
+**Please note that non-LTS versions of Node and npm are not supported. **
```

Comment:
I'm afraid [npm 7 already reached GA](https://github.blog/2021-02-02-npm-7-is-now-generally-available/) so we need to improve this message a bit 😅 

```suggestion
**Generator is tested at the moment against Node 14 and NPM 6. Using newer versions is enabled but we do not guarantee they work well. Please provide feedback on the issues.**

how about this?
```